tests/service/views/test_cache_views.py

```diff
@@ -67,7 +67,7 @@ def test_list_upload_files_all_no_auth(svc_client):
     assert 200 == response.status_code
 
     assert {"error"} == set(response.json.keys())
-    assert INVALID_HEADERS_ERROR_CODE == response.json["error"]["code"]
+    assert UserAnonymousError().code == response.json["error"]["code"]
```

Comment:
I don't think you need to instantiate the error to get the class variable, it should work without brackets.

```suggestion
    assert UserAnonymousError.code == response.json["error"]["code"]
```

Reply:
Oh... you are right... let me adjust the 51 occurrences of `().code` :sweat_smile: